functions

```diff
@@ -71,3 +72,7 @@ pass() {
 fail() {
   echo $'\x1B[0;31m\xE2\x9C\x98\x1B[0m' "$@"
 }
+
+node_major_version() {
+  node -p process.versions.node | cut -d'.' -f1
```

Comment:
I don't know why it's so common to see option names and values squashed together, for `cut` in particular.

Let's give the arguments some breathing room:

```suggestion
  node --print process.versions.node | cut -d . -f 1
```